kornia/color/rgb.py

```diff
@@ -17,6 +17,12 @@ class RgbToBgr(nn.Module):
         - image: :math:`(*, 3, H, W)`
         - output: :math:`(*, 3, H, W)`
 
+    Examples::
+
+        >>> input = torch.rand(2, 3, 4, 5)
+        >>> bgr = kornia.image.RgbToBgr()
```

Comment:
This is kornia.color

Reply:
excuse me, I didn't understand the request. I just added the example form in every class in kornia.color

Reply:
I think I got it, you mean instead of kornia.image.... it'll be kornia.color? sorry my bad I'll fix it